### ISSUE TITLE
Aggregation Layer

### ISSUE THREAD

**Issue by jduerholt** [atomistic-machine-learning/schnetpack#159]
Dear schnetpack developers,

I am wondering why it is necessary to have always an aggregation layer in an Atomwise output module? There could be the case that aggregation is not needed and one is only interested in  atomic properties. 

An example could be the training of atomic charges with respect to reference charges computed by any population analysis. In this case no aggregation would be necessary. But for me it looks like you have to always specify an aggregation type. Of course it could be that I overlooked something and there is already a possibility for this.

Best

Johannes

**Comment by ktschuett** [comment]
Hi,
you are right, there is not explcit layer at the moment for output per atom. However, you can use the Atomwise layer for that as follows:
```
output = Atomwise(n_in, contributions="charges")
```
The result dict will contain the atomwise property "charges" then.

**Comment by jduerholt** [comment]
Hi,

thanks for your response. I also thought about this opportunity but I did not like that it is then also calculating the average. What I did was to add an additional "squeeze" option to the Aggregation layer:

`if self.squeeze == True:
     y = input.squeeze()
     return y
`
Since the atom contributions are always of the shape (batchsize, natoms, 1), the squeeze option reduces it to (batchsize, natoms) which makes for me more sense for fitting charges. In addition, I added to the `extxyz_to_db` method the option for writing charges to the ase db file out of a extxyz file.

I know this changes are very trivial, but if you are interested, I can make a pull request.

Best

Johannes
